src/Raven.Server/Smuggler/Documents/Actions/DatabaseRecordActions.cs

```diff
@@ -88,7 +88,7 @@ public DatabaseRecordActions(ServerStore server, DatabaseRecord currentDatabaseR
             result.DatabaseRecord.ConflictSolverConfigUpdated = true;
         }
 
-        if (databaseRecord.PeriodicBackups.Count > 0 && databaseRecordItemType.HasFlag(DatabaseRecordItemType.PeriodicBackups))
+        if (CanAccess(authorizationStatus) && databaseRecord.PeriodicBackups.Count > 0 && databaseRecordItemType.HasFlag(DatabaseRecordItemType.PeriodicBackups))
```

Comment:
We need to alert on those , no?

Reply:
![image](https://github.com/ravendb/ravendb/assets/18046191/8fa3c32a-21e1-4f30-8486-afbfee029b25)